Assets/Scripts/FpsController.cs

```diff
@@ -0,0 +1,178 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FpsController : MonoBehaviour
+{
+    [Header("Camera Settings")]
+    public Camera cam;
+    public float lookHorizontalSensitivity = 1.0f;
+    public float lookVerticalSensitivity = 1.0f;
+    public float minimumPitch = -89.9f;
+    public float maximumPitch = 89.9f;
+    private float _lookYaw;
+    private float _lookPitch;
+
+    [Header("Jump Settings")]
+    public float jumpForce = 300.0f;
+    public float jumpInputBufferTime = 0.25f;
+    [SerializeField]
+    private bool _jumpInput = false;
+    [SerializeField]
+    private float _jumpTimeSinceInput = 0.0f;
+
+    [Header("Movement Settings")]
+    [Range(0.1f, 15.0f)]
+    public float forwardSpeed = 5.0f;
+    [Range(0.1f, 15.0f)]
+    public float backSpeed = 3.0f;
+
+
+    [SerializeField]
```

Comment:
nit: remove double linebreak

```suggestion

    [SerializeField]
```